src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
Done. The data are not exactly the same: the colors are different types (SColor vs SColorf), and height and thickness were multiplied by BS in Clouds but not in CloudSettings, but I've corrected for that.

Also: everything is called CloudParameters or cloud_parameters or CLOUD_PARAMETERS, hope that's ok.